core/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -423,36 +423,26 @@ public void deleteSnapshot(SnapshotId snapshotId, long repositoryStateId) {
     }
 
     private void safeSnapshotBlobDelete(final SnapshotInfo snapshotInfo, final String blobId) {
```

Comment:
Maybe rename it into `deleteSnapshotBlobIgnoringErrors` while you are at it?

Reply:
Sure